### ISSUE TITLE
vscode is broken

### ISSUE THREAD

**Issue by jgillich** [NixOS/nixpkgs#14890]
## Issue description
VS Code installed from master doesn't run.

```
$ code
code: error while loading shared libraries: libpangocairo-1.0.so.0: cannot open shared object file: No such file or directory
```

@abbradar I saw you made some changes to atom and removed buildEnv, maybe that also needs to be done here? vscode uses a copy of the atom buildEnv.

Also cc @mucaho 

### Steps to reproduce
Install `vscode` from master.

## Technical details
* System: NixOS unstable

**Comment by abbradar** [comment]
Try [this patch](https://github.com/abbradar/nixpkgs/commit/c33afce5685bd19b6f9e0ddea98dff41d578fd9d) and see if it works.

**Comment by jgillich** [comment]
Thanks, that works! However:

```
nix-env -f ~/devel/nixpkgs/ -iA pkgsi686Linux.vscode
replacing old ‘vscode-1.0.0’
installing ‘vscode-1.0.0’
error: Package ‘atom-1.6.2’ in ‘/home/jakob/devel/nixpkgs/pkgs/applications/editors/atom/default.nix:58’ is not supported on ‘i686-linux’, refusing to evaluate.
a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build` or any other Nix command you can add
  { allowBroken = true; }
to ~/.nixpkgs/config.nix.

(use ‘--show-trace’ to show detailed location information)
```

**Comment by abbradar** [comment]
Ugh, not nice. I'll try to invent some other way... #11094 would help, too.

**Comment by abbradar** [comment]
See my branch https://github.com/abbradar/nixpkgs/commits/vscode-electron -- it should fix this and also unify and cleanup things for all three packages a bit.

**Comment by jgillich** [comment]
Awesome! I included it in #14891
